### PR TITLE
adding in a slug on "deleting" == merging for grid layouts

### PR DESCRIPTION
**Pull request by crutkas**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/fancyzones.md [hunk 1]
```diff
@@ -79,9 +79,9 @@ There are two ways to create custom zone layouts: **Grid** layout and **Canvas**
 
 The **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired. This is a relative layout and will resize with different screen sizes.
 
-- To divide a zone, just click your mouse.  To rotate the divider, just hold down the shift key. 
+- To divide a zone, just click your mouse.  To rotate the divider, just hold down the <kbd>shift</kbd> key. 
 - To move a divider, click on the thumb and drag.
-- To merge zones, select and hold the left mouse button and drag the mouse until multiple zones are selected, then release the button and a popup menu will show up.  Click "Merge" and they will become one big zone.
+- To merge zones, select and hold the left mouse button and drag the mouse until multiple zones are selected, then release the button and a popup menu will show up.  Click "Merge" and they will become one big zone. This is how you'd a zone would be deleted, it is merge it into another zone.
```
Comment (pauby) [inline hub/powertoys/fancyzones.md]:
```suggestion
- To merge zones, select and hold the left mouse button and drag the mouse until multiple zones are selected, then release the button and a popup menu will show up.  Click "Merge" and they will become one big zone. This is how a zone would be deleted, by merging it into another zone.
```

**Comment by franky920920** [comment]
This resolves #3626 and https://github.com/microsoft/PowerToys/issues/15598

**Comment by PRMerger18** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by crutkas** [comment]
@mattwojo this is good to go if you are

**Pull request merged**